tuxemon/core/event/actions/variable_math.py

```diff
@@ -49,12 +49,14 @@ def start(self):
 
         # Read the parameters
         var = self.parameters.var
-        operand1 = number_or_variable(game, var)
+        operand1 = number_or_variable(self.game, var)
         operation = self.parameters.operation
-        operand2 = number_or_variable(game, self.parameters.value)
+        operand2 = number_or_variable(self.game, self.parameters.value)
 
         # Preform the operation on the variable
-        if operation == "+":
+        if operation == "=":
```

Comment:
please use `==` so that the operations are similar to python and other langauges

Reply:
This is the event action though, not the conditional: It's setting the variable rather than checking it. This should mirror the common standard, as = is used to set and == is used to check.

Reply:
ah good point.  i had though this was comparison.